src/index.ts

```diff
@@ -270,7 +243,10 @@ function isTestnet(networkName: string): boolean {
  */
 function getGatewayUrl(args: any, hre: HardhatRuntimeEnvironment): string {
     let gatewayUrl: string = args.gatewayUrl;
-    const networkName: string = args.starknetNetwork || process.env.STARKNET_NETWORK;
+    let networkName: string = args.starknetNetwork || process.env.STARKNET_NETWORK;
+    if (isMainnet(networkName)) {
+        networkName = ALPHA_MAINNET_INTERNALLY;
+    }
 
```

Comment:
@dribeiro-ShardLabs During the rebase, I guess I should have added an `isTestnet` check here; right?

Reply:
Yes, so we use our internal naming for testnet as well